storage/Volume.cc

```diff
@@ -2441,7 +2441,11 @@ namespace storage
     string Volume::getDmcryptName() const
     {
 	string nm;
-	if( !mp.empty() )
+        if ( mp == "/" )
```

Comment:
Yes. It's broken in more places in that file, but our coding style actually demands a blank between `if` and the opening parenthesis.